lib/services/oauth/oauth_server_test.go

```diff
@@ -0,0 +1,79 @@
+package oauth_test
+
+import (
+	"net/http"
+
+	"github.com/markgravity/golang-ic/lib/services/oauth"
+	"github.com/markgravity/golang-ic/test"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("OauthServer", func() {
+	Describe("HandleTokenRequest", func() {
+		Context("Given VALID params", func() {
+			It("returns status OK", func() {
```

Comment:
```suggestion
			It("responds with the OK status", func() {
```

The main idea is just to avoid having 2 tests that asses the same attribute ("it returns ...") with a different value.
When I have 1 request, it returns A or B, but it cannot return A and return B. ;-)